plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -46,12 +48,34 @@ const uint16_t EuroliteProFactory::VENDOR_ID_MK2 = 0x0403;
 
 const char EuroliteProFactory::ENABLE_EUROLITE_MK2_KEY[] =
     "enable_eurolite_mk2";
+const char EuroliteProFactory::EUROLITE_MK2_SERIAL_KEY[] =
+    "eurolite_mk2_serial";
 
 EuroliteProFactory::EuroliteProFactory(ola::usb::LibUsbAdaptor *adaptor,
                                        Preferences *preferences)
   : BaseWidgetFactory<class EurolitePro>("EuroliteProFactory"),
     m_adaptor(adaptor),
     m_enable_eurolite_mk2(IsEuroliteMk2Enabled(preferences)) {
+  const std::vector<std::string> serials =
+      preferences->GetMultipleValue(EUROLITE_MK2_SERIAL_KEY);
+  // A single empty string is considered the same as specifying
+  // no serial numbers. This is useful as a default value.
+  const bool has_default_value =
+      serials.size() == 1 && serials[0].empty();
+  if (!has_default_value) {
+    for (std::vector<std::string>::const_iterator iter = serials.begin();
+         iter != serials.end(); ++iter) {
+      if (iter->empty()) {
+        OLA_WARN << EUROLITE_MK2_SERIAL_KEY
+                 << " requires a serial key string, but it is empty.";
```

Comment:
Key is just our internal term for the key part of the key-value setting.
```suggestion
                 << " requires a serial number, but it is empty.";
```